File: WindowsApp3a/Form1.cs
using System;
using System.IO;
using System.Windows.Forms;
using WindowsApp3a.Classes;

namespace WindowsApp3a
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void exportExcelButton_Click(object sender, EventArgs e)
        {
            string fileName = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Customers.xlsx");
            var rowsExported = 0;

            var ops = new DataOperations();
            ops.WriteCustomerTableToXmlFile(fileName, ref rowsExported);

            MessageBox.Show(ops.IsSuccessFul ? $"Done, exported {rowsExported} rows" : $"Error{Environment.NewLine}{ops.LastExceptionMessage}");
        }
    }
}

File: WindowsApp3/Form1.vb
Imports System.IO
Imports WindowsApp3.Classes

Public Class Form1
    Private Sub exportExcelButton_Click(sender As Object, e As EventArgs) Handles exportExcelButton.Click

        Dim fileName As String = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Customers.xlsx")
        Dim rowsExported = 0

        Dim ops As New DataOperations
        ops.WriteCustomerTableToXmlFile(fileName, rowsExported)

        If ops.IsSuccessFul Then
            MessageBox.Show($"Done, exported {rowsExported} rows")
        Else
            MessageBox.Show($"Error{Environment.NewLine}{ops.LastExceptionMessage}")
        End If

    End Sub
End Class
